configuration/configuring-root-certificates/en.md

```diff
@@ -0,0 +1,119 @@
++++
+title = "Configuring Root Certificates"
+lastmod = "2022-06-10T14:22:00-06:00"
++++
+# Configuring Root Certificates
+
+Mange local SSL/TLS Certificate Authority (CA) certificates with any of the desktop environments supported by Solus.
+
+## Public vs. Local Certificates
+
+Public SSL certificates are included in Solus system packages and are regularly updated with normal package updates.
+
+There are many reasons you may want to install a certificate that is custom, self-signed, or provided by your local systems administrator.
+
+The topic of SSL and TLS certificates for protecting network resources is a deep and complex topic. Many great resources are publicly available for fundamental usage and workflows. This article will describe the default utilities and NSS database location in Solus.
+
+## Basic Requirements
+
+This article assumes basic familiarity with using the command line. 
+
+A web browser and desktop environment like Budgie can be very useful in testing a certificate served for a network resource.
+
+The creation and validation of a certificate is outside the scope of this article and you must already have a signed certificate. 
+
+These instructions assume usage for secure network connections on the internet or corporate networks, but may also be used for other purposes like signing e-mail.   
+
+{{< altimg "network-overview.jpg" "help-center/configuration/configuring-network/" >}}
+
+WARNING: You must trust the source of any certificate to be installed. Using an insecure or otherwise compromised certificate is a critical security risk.
+
+## The NSS Database
+
+Solus utilizes Network Security Service(NSS) for managing digital certificates. Each system user account will have a user NSS database located at `$HOME/.pki/nssdb`.   
+
+The `certutil` utility is installed by default and can be verified with `which certutil`. We will be operating on the logged in users' NSSDB, without need for Root permissions. A summary of options can be shown with `--syntax`, and `-H` will show a complete list of option descriptions.
```

Comment:
```suggestion
The `certutil` utility is installed by default and can be verified with `which certutil`. We will be operating on the logged-in user's NSSDB, without need for Root permissions. A summary of options can be shown with `--syntax`, and `-H` will show a complete list of option descriptions.
```

since we're talking about a single user (I assume), and my preferred spelling of "logged-in"